go.mod

```diff
@@ -99,7 +99,7 @@ require (
 	github.com/cockroachdb/tokenbucket v0.0.0-20230807174530-cc333fc44b06 // indirect
 	github.com/coinbase/rosetta-sdk-go/types v1.0.0 // indirect
 	github.com/cosmos/gogogateway v1.2.0 // indirect
-	github.com/cosmos/iavl v1.1.3 // indirect
+	github.com/cosmos/iavl v1.2.0 // indirect
```

Comment:
Checked. Can we bump to 1.3.0?

Reply:
I'll do that in a separate PR if that's ok?